### PR TITLE
Regenerating the Internal CA for Ingress

### PR DESCRIPTION
**Pull request by ShudiLi**
@lihongan @liangxia @jhou1  @xingxingxia @wangke19  Please help review the ingress part, thanks!

### CODE REVIEW

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh [hunk 1]
```diff
@@ -83,3 +83,56 @@ oc adm ocp-certificates remove-old-trust -n openshift-kube-apiserver-operator co
...
+proxyCert="temProxy.pem"
+cusIngressKey="wilcard.key"
+cusIngressCsr="wildcard.csr"
+cusIngressCert="wildcard.pem"
+keyLength=4096
+baseDomin=$(oc get dns.config cluster -o=jsonpath='{.spec.baseDomain}')
+defaultIngressDomin=$(oc get ingresscontroller default  -o=jsonpath='{.status.domain}' -n openshift-ingress-operator)
+currentPath=$(pwd)
+workPath="/tmp/replcertforingress"
+sudo mkdir $workPath; sudo chmod 664 $workPath; cd $workPath
```
Comment (lihongan) [inline ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh:98]: I hit error `error: error reading /tmp/replcertforingress/temProxy.pem: permission denied` 
please check if chmod related

**Comment by lihongan** [comment]
@ShudiLi please check the failed job https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/40789/rehearse-40789-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-stable-4.13-azure-ipi-fips-cert-rotation-p2-f14/1674291937227575296

the error 
```
 ++ pwd
+ currentPath=/
+ workPath=/tmp/replcertforingress
+ sudo mkdir /tmp/replcertforingress
/bin/bash: line 99: sudo: command not found 
```

please fix the script issue. thanks

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh (hunk 1 above)
Comment (ShudiLi) [reply ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh:98]: From error /bin/bash: line 99: sudo: command not found in the below review comment， so sudo mkdir $workPath doesn't work, I will remove all sudo commands, thanks.

**Comment by lihongan** [comment]
@ShudiLi still seeing new scripts issue 
```
 + workPath=/tmp/replcertforingress
+ mkdir /tmp/replcertforingress
+ chmod 664 /tmp/replcertforingress
+ cd /tmp/replcertforingress
/bin/bash: line 99: cd: /tmp/replcertforingress: Permission denied 
```

please fix the script issue. thanks

**Comment by lihongan** [comment]
/retest

job failed before the ingress cert regeneration 
```
 All clusteroperators are stable
+ oc config new-admin-kubeconfig
+ oc --kubeconfig=/tmp/admin.kubeconfig whoami
error: You must be logged in to the server (Unauthorized) 
```

**Comment by lihongan** [comment]
/pj-rehearse ack

since the step passed in some jobs
```
INFO[2023-06-30T19:48:59Z] Running step azure-ipi-fips-cert-rotation-p2-f1-openshift-upgrade-qe-test-cert-rotation. 
INFO[2023-06-30T21:49:29Z] Step azure-ipi-fips-cert-rotation-p2-f1-openshift-upgrade-qe-test-cert-rotation succeeded after 2h0m30s.

 INFO[2023-06-30T19:00:28Z] Running step aws-ipi-byo-iam-role-cert-rotation-p2-f14-openshift-upgrade-qe-test-cert-rotation. 
INFO[2023-06-30T21:14:18Z] Step aws-ipi-byo-iam-role-cert-rotation-p2-f14-openshift-upgrade-qe-test-cert-rotation succeeded after 2h13m49s.
```
the failures in other jobs are unrelated to the changes of this PR

**Comment by liangxia** [comment]
For all the tests that passed step `*-openshift-upgrade-qe-test-cert-rotation`, the next step `*-openshift-extended-upgrade-post` failed with `Unable to connect to the server: tls: failed to verify certificate: x509: certificate signed by unknown authority`
@kuiwang02 Could you help find someone to check deeper for the failure ?

/hold

**Comment by lihongan** [comment]
```
Unable to connect to the server: x509: certificate signed by unknown authority
```
it should be the bug https://issues.redhat.com/browse/OCPBUGS-15581 reported by @wangke19 
more details see thread: https://redhat-internal.slack.com/archives/C05ANRBDYLF/p1687916726627379

**Pull request merged**